### DISCUSSION TITLE
Does the wallet-core NPM library even work?

### DISCUSSION THREAD

**Discussion by ethanmarcuss** [trustwallet/wallet-core#2253] [Q&A] [answered]
Hey everyone!

I'm building a multichain wallet and would love to utilize Wallet-Core. I'm building with React Native and need a JS multichain wallet SDK. I know Wallet Core has an NPM package but there is no documentation on it and it's listed as a "Beta" release.

Is it still possible to build with it?

Thanks!

**Comment by hewigovens** [comment, marked as answer]
Yes, you can see these tests https://github.com/trustwallet/wallet-core/tree/master/wasm/tests, we haven't updated docs yet

**Reply by ethanmarcuss** [reply]
That's awesome, thank you so much! Can't wait to get building.
